types/parameter/index.d.ts

```diff
@@ -0,0 +1,247 @@
+// Type definitions for parameter 3.6
+// Project: https://github.com/node-modules/parameter#readme
+// Definitions by: Innpaul Zhang <https://github.com/zhyupe>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
+
+export = parameter;
+
+declare class parameter {
+    constructor(options?: parameter.ParameterOptions);
+
+    /**
+     * Add custom rules
+     * @param type
+     * @param check
+     * @param override Override exist rules. Default: `true`
+     */
+    addRule: typeof parameter.addRule;
+
+    /**
+     * Translate function
+     */
+    t: parameter.ParameterTranslateFunction;
+
+    /**
+     * Validate the value conforms to rule. return an array of errors if break rule.
+     * @param rule
+     * @param value
+     */
+    validate(rule: parameter.ParameterRules, value: unknown): parameter.ValidateError[] | void;
+
+    static CONVERT_MAP: Record<string, parameter.ParameterConvertType | undefined>;
+
+    /**
+     * Add custom rules
+     * @param type
+     * @param check
+     * @param override Override exist rules. Default: `true`
+     */
+    static addRule<T extends string>(
```

Comment:
it seems there should be a version with 3 parameters:
https://github.com/node-modules/parameter/blob/master/index.js#L155

Reply:
Fixed in 6d5343969e588e21e35fff8ce959946b1669e48d